wikimedia/wikipedia-ios
# [ALT TEXT] Add ability to send analytics to wmflabs

**Phabricator:** https://phabricator.wikimedia.org/T371412

### Test Steps
1. Fresh install app. Go to developer settings menu and enable "Send analytics to wmflabs"
2. Open https://stream-beta.wmflabs.org/v2/ui/#/ and pick a stream to watch
3. Trigger events in the app from the stream. Confirm after some time you see them appear in the list.
